src/core/stdc/stdio.d

```diff
@@ -1058,14 +1058,14 @@ else version (Solaris)
         _IOMYBUF = 0x08,
     }
 
-    private extern shared FILE[_NFILE] __iob;
+    private extern __gshared FILE[_NFILE] __iob;
 
     ///
-    shared stdin = &__iob[0];
+    @property auto stdin()() { return &__iob[0]; }
```

Comment:
Hmm, that's a `FILE*` now, not a `shared(FILE)*`, and accessing it probably not `@safe`. Let's keep `shared`?

Reply:
`FILE` itself is a `shared` alias, so it doesn't make a difference.

Reply:
See https://d.godbolt.org/z/j14jW9svn

Reply:
Oh, okay. But the `@safe` breaking change remains: https://d.godbolt.org/z/ccx7s59c1

Reply:
Good catch, I'll remove all `__gshared` from all extern FILE declarations.

Reply:
Done.